### PR TITLE
Add a min and max var to collection types

### PR DESCRIPTION
**Pull request by bjorvack**
Add min/max data attributes to collection types

### CODE REVIEW

**Comment by tijsverkoyen** [comment]
Why no server side validation? As in:
* less then minimum options == error
* more than maximum options == error

File: src/Form/Extension/CollectionTypeExtension.php [hunk 1]
```diff
@@ -24,6 +24,8 @@ public function configureOptions(OptionsResolver $resolver): void
             [
                 'allow_drag_and_drop' => true,
                 'add_button_label' => 'forms.buttons.addItem',
+                'min' => 0,
+                'max' => null,
```
Comment (tijsverkoyen) [inline src/Form/Extension/CollectionTypeExtension.php:28]: I would call this `maximum_allowed_items`

Comment (bjorvack) [reply src/Form/Extension/CollectionTypeExtension.php]: fixed [dd884eb](https://github.com/sumocoders/FrameworkCoreBundle/pull/138/commits/dd884eb960aaceed002f0d33c351ffce59253eb4)

File: src/Form/Extension/CollectionTypeExtension.php [hunk 2]
```diff
@@ -24,6 +24,8 @@ public function configureOptions(OptionsResolver $resolver): void
             [
                 'allow_drag_and_drop' => true,
                 'add_button_label' => 'forms.buttons.addItem',
+                'min' => 0,
```
[the remark this answers is not in the document]
Comment (bjorvack) [reply src/Form/Extension/CollectionTypeExtension.php]: fixed [dd884eb](https://github.com/sumocoders/FrameworkCoreBundle/pull/138/commits/dd884eb960aaceed002f0d33c351ffce59253eb4)

**Comment by bjorvack** [comment]
@tijsverkoyen added validation in [0a522e6](https://github.com/sumocoders/FrameworkCoreBundle/pull/138/commits/0a522e6e7490daa9b48cf9f9eb04620037905aef)

**Review by tijsverkoyen — APPROVED** [review]

**Pull request merged**